### PR TITLE
Item app [draft]

### PR DESCRIPTION
**Pull request by AleksanderPawlak**

### CODE REVIEW

File: coffee_roaster_market/apps/conftest.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import pytest
```
Comment (AleksanderPawlak) [inline coffee_roaster_market/apps/conftest.py:1]: I didn't know if it's good idea to place global conftest here, but in my opinion implementing a `client` fixture for each app or importing it from `apps.coffee.tests` would be weird. @kuchichan what do you think about that?

Comment (kuchichan) [reply coffee_roaster_market/apps/conftest.py:1]: I think it's fine.

File: coffee_roaster_market/apps/item/models.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from coffee_roaster_market.apps.coffee.models import CoffeeModel
+
+
+class ItemModel(models.Model):
+    class Meta:
+        db_table = "item"
+
+    coffee = models.OneToOneField(CoffeeModel, on_delete=models.CASCADE)
```
Comment (kuchichan) [inline coffee_roaster_market/apps/item/models.py]: As mentioned during phone call - `ForeignKey` will be more sufficient (Many to one relationship)

**Review by kuchichan — CHANGES_REQUESTED** [review]
Just a relationship needs to be refined.

File: coffee_roaster_market/apps/item/models.py (hunk 2 above)
Comment (AleksanderPawlak) [reply coffee_roaster_market/apps/item/models.py]: Done

**Review by kuchichan — APPROVED** [review]

**Pull request merged**